commit 907b91e7154d57f091af547aa2179e1433bc69eb
Author: Gao,Yan <ygao@suse.com>
Date:   Thu Apr 18 16:00:02 2013 +0800

    Log: fencing: Drop the severity of the messages on registering a stonith device

Index: pacemaker-2.1.6+20231030.66cc0f083/daemons/fenced/fenced_commands.c
===================================================================
--- pacemaker-2.1.6+20231030.66cc0f083.orig/daemons/fenced/fenced_commands.c
+++ pacemaker-2.1.6+20231030.66cc0f083/daemons/fenced/fenced_commands.c
@@ -1465,7 +1465,7 @@ stonith_device_register(xmlNode *dev, gb
         g_hash_table_replace(device_list, device->id, device);
 
         ndevices = g_hash_table_size(device_list);
-        crm_notice("Added '%s' to device list (%d active device%s)",
+        crm_info("Added '%s' to device list (%d active device%s)",
                    device->id, ndevices, pcmk__plural_s(ndevices));
     }
 
Index: pacemaker-2.1.6+20231030.66cc0f083/daemons/fenced/fenced_cib.c
===================================================================
--- pacemaker-2.1.6+20231030.66cc0f083.orig/daemons/fenced/fenced_cib.c
+++ pacemaker-2.1.6+20231030.66cc0f083/daemons/fenced/fenced_cib.c
@@ -618,7 +618,7 @@ update_cib_cache_cb(const char *event, x
                 break;
             case -pcmk_err_diff_resync:
             case -pcmk_err_diff_failed:
-                crm_notice("[%s] Patch aborted: %s (%d)", event, pcmk_strerror(rc), rc);
+                crm_info("[%s] Patch aborted: %s (%d)", event, pcmk_strerror(rc), rc);
                 free_xml(local_cib);
                 local_cib = NULL;
                 break;
